src/main/java/uk/gov/hmcts/darts/common/entity/RegionEntity.java

```diff
@@ -28,4 +34,10 @@ public class RegionEntity {
     @Column(name = "region_name")
     private String regionName;
 
+    @ManyToMany
+    @JoinTable(name = "courthouse_region_ae",
```

Comment:
Please check this association - pick a side to own the relationship.
So that the other side only needs to know the mappedBy, as it looks like the joinColumns / inverseColumns are wrong here to me - copy / paste issue.
https://github.com/search?q=repo%3Ahmcts%2Fdarts-api%20%40ManyToMany(mappedBy&type=code